src/services/user-service.js

```diff
@@ -22,7 +22,18 @@ function logout() {
   localStorage.removeItem('user');
 }
 
-function register(user) {}
+function register(username, password, firstName, lastName) {
+  const requestOptions = {
+    method: 'POST',
+    headers: { 'content-Type': 'application/json' },
+    body: JSON.stringify({ firstName, lastName, username, password }),
+  };
+  return fetch('/users/register', requestOptions)
+    .then(handleResponse)
+    .then((user) => {
```

Comment:
you can get rid of lines 33-35. This will cause you application to reroute to the login page where you will be able to login with your new credentials